test/unit/ticker.c

```diff
@@ -64,10 +64,32 @@ TEST_BEGIN(test_ticker_copy) {
 }
 TEST_END
 
+TEST_BEGIN(test_ticker_geom) {
+	const int32_t ticks = 100;
+	const uint64_t niters = 100 * 1000;
+
+	ticker_geom_t ticker;
+	ticker_geom_init(&ticker, ticks);
+	uint64_t total_ticks = 0;
+	/* Just some random constant. */
+	uint64_t prng_state = 0x343219f93496db9fULL;
+	for (uint64_t i = 0; i < niters; i++) {
+		while(!ticker_geom_tick(&ticker, &prng_state)) {
+			total_ticks++;
+		}
+	}
+	expect_u64_ge(total_ticks / niters, ticks * 9 / 10,
```

Comment:
nit: add comment to include a typical result; usually the mean should be very close?

Reply:
Done.

Really, we quantize coarsely enough that there is some bias in the resulting output, but shrinking that by making the ticker table bigger didn't seem worth it to me (we don't really care if the average tick time is exactly 1000 or 1050 or whatever).